README.md

```diff
@@ -88,9 +88,7 @@ docker-compose up
 
 ### Setting up the PCEX backend
 
-The PCEX backend isn't affected by CORS, so you don't have to run a special one. 
-
-At the time of writing this README, you need two seperate instances of the PCEX API - one 3.4.0 instance to do indicator access and one 3.5.0 instance to do watershed analysis, due to a weird rasterio conflict, but this should be very temporary. 
+The PCEX backend isn't affected by CORS, so you don't have to run a special one; any PCEX backend with spare capacity (IE, not production) will work.
```

Comment:
As discussed in our call, we can drop all these sections about backends and just leave it to the configuration section below. It looks like the comments there identify which backends are being pointed at, but we could conceivably list them out explicitly beforehand, compressing these CORS sections into 3 lines or so.